spec/lib/open_weather/models/station_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe OpenWeather::Models::Station do
```

Comment:
Probably shorter to use the same VCR here. The problem with the double is that it will continue working even if we break the API call.

Reply:
Curious, since that client method is unit-tested already, wouldn't it be sufficient to unit-test this as well (check that we call client with the right params)?

Reply:
It's sufficient, but implies behavior and IMHO hides bugs. I've debugged my fair share of `double`'s to avoid them, more often than to use them. That said no strong opinions.